barretenberg/cpp/src/barretenberg/stdlib/honk_recursion/verifier/protogalaxy_recursive_verifier.test.cpp

```diff
@@ -97,31 +105,25 @@ template <typename RecursiveFlavor> class ProtoGalaxyRecursiveTests : public tes
         fq_ct big_b(fr_ct(witness_ct(&builder, bigfield_data_b.to_montgomery_form())), fr_ct(witness_ct(&builder, 0)));
 
         big_a* big_b;
-
```

Comment:
Why did you remove additional gates for Goblin?

Reply:
because big_a * big_b should already create goblin ops and that should be enough for testing purposes